docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -6,11 +6,11 @@ weight = 128
 
 # Enable Grafana 8 Alerts
 
-Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
+Grafana 8 alerting system is enabled by default.
 
-> **Note:** We recommend that you backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
+> **Note:** We recommend that you backup Grafana's database before upgrading to the next 8.2 release. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
```

Comment:
This note should be moved to https://grafana.com/docs/grafana/latest/alerting/unified-alerting/ topic and also to https://grafana.com/docs/grafana/latest/alerting/, so that users can see this upfront.